lib/Fencer/Logic.hs

```diff
@@ -2,20 +2,19 @@
 {-# LANGUAGE OverloadedLabels #-}
 {-# LANGUAGE RecordWildCards #-}
 
--- | In-memory state of Fencer.
-module Fencer.AppState
+-- | In-memory state of Fencer and the logic for managing it.
+module Fencer.Logic
     ( AppState
-    , appStateCounters
     , appStateRules
```

Comment:
Can you also remove this export? It's not used anywhere else in the code and it allows breaking internal invariants.

Reply:
I guess I introduced it at some point and forgot to remove it. Thank you for spotting it!

This is taken care of in an upcoming commit.